app/rust-ffi/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 use wasm_bindgen::prelude::*;
 
+use enso_parser::lexer::test::TokenOperatorProperties;
```

Comment:
That's a test reexport, the proper path is something like `enso_parser::syntax::token::TokenOperatorProperties`

Reply:
Ah, auto-imports in IDE…